engine/src/main/scala/cromwell/engine/workflow/lifecycle/finalization/CopyWorkflowOutputsActor.scala

```diff
@@ -69,19 +82,33 @@ class CopyWorkflowOutputsActor(workflowId: WorkflowId, override val ioActor: Act
   }
   
   private def getOutputFilePaths(workflowOutputsPath: Path): List[(Path, Path)] = {
+
+    val flattenWorkflowOutputs: Boolean = workflowDescriptor.getWorkflowOption(FlattenWorkflowOutputs) match {
+      case Some("true") => true
+      case _ => false
+    }
     val rootAndFiles = for {
       // NOTE: Without .toSeq, outputs in arrays only yield the last output
       backend <- workflowDescriptor.backendAssignments.values.toSeq
       config <- BackendConfiguration.backendConfigurationDescriptor(backend).toOption.toSeq
       rootPath <- getBackendRootPath(backend, config).toSeq
       outputFiles = findFiles(workflowOutputs.outputs.values.toSeq).map(_.value)
     } yield (rootPath, outputFiles)
-    
+
+    // This regex will make sure the path is relative to the execution folder.
+    // the call-.* part is there to prevent arbitrary folders called execution to get caught.
+    val truncateRegex = ".*/call-.*/execution/".r
```

Comment:
I would move this inside the `if` or make it `lazy` so that it only gets computed if it's needed

Reply:
Inside the if it will be calculated for every file. Which is a major performance hit. Making it lazy is indeed much better. Thanks!